tests/nanoFramework.WebServer.Tests/WebServerTests.cs

```diff
@@ -0,0 +1,79 @@
+using nanoFramework.TestFramework;
+using System;
+
+namespace nanoFramework.WebServer.Tests
+{
+    [TestClass]
+    public class WebServerTests
+    {
+        // IsRouteMatch GET, POST, PUT, DELETE, PATCH
+        // IsRouteMatch CaseSensitive true/false
+
+        [TestMethod]
+        public void IsRouteMatch_Should_ReturnFalseForEmptyMethod()
+        {
+            // Arrange
+            var route = new CallbackRoutes()
+            {
+                Method = "GET",
+                Route = "/api/test"
+            };
+
+            // Act
+            var result = WebServer.IsRouteMatch(route, "", "/api/test");
+
+            // Assert
+            Assert.IsFalse(result);
+        }
+        
+        [TestMethod]
+        public void IsRouteMatch_Should_ReturnFalseForNotMatchingMethod()
+        {
+            // Arrange
+            var route = new CallbackRoutes()
+            {
+                Method = "GET",
+                Route = "/api/test"
```

Comment:
and here as well

Reply:
Here i'm testing if the method will return early for empty method name. I don't thing query params here are required.